ScanbotSDKExamples/MAUI/ClassicComponent.Maui/Platforms/iOS/AppDelegate.cs

```diff
@@ -37,5 +39,48 @@ string GetDemoStorageBaseDirectory()
 
         return folder;
     }
+
+
+    /// <summary>
+    /// Show message on top of the Root window
+    /// </summary>
+    /// <param name="message"></param>
+    /// <param name="buttonTitle"></param>
+    internal void ShowAlert(string message, string buttonTitle)
+    {
+        var alert = UIAlertController.Create("Alert", message, UIAlertControllerStyle.Alert);
+        var action = UIAlertAction.Create(buttonTitle ?? "Ok", UIAlertActionStyle.Cancel, (obj) => { });
+        alert.AddAction(action);
+        Window?.RootViewController?.PresentViewController(alert, true, null);
+    }
+
+    /// <summary>
+    /// Extract ViewController from the Application's ViewController Hierarchy.
+    /// </summary>
+    /// <returns></returns>
+    internal static UIViewController ExtractViewController(UIWindow window)
+    {
+        var viewController = window?.RootViewController;
+
+        if (viewController == null) return null;
+        
+        // If application has a Navigation Controller
+        if (viewController is UINavigationController navigationController)
+        {
+            // ALERT: In case of Navigation Controller we are explicitely accessing the ChildViewController and then returning the last object in stack.
+            // Because there is an issue OR we can say it is built this way in MAUI.
```

Comment:
@scan-mayank, I think we can leave out this line (about there being an issue or being built this way) because we don't know, and we already describe why we cannot do it the simple way.